content/blog/handling-cheapest-fuel-data/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Handling the Cheapest Fuel- Data"
+date: "2020-10-06"
+author: "Samriddhi Mishra"
+coverImage: "a.png"
+tags: ["Data", "BigData", "Hadoop"]
+description: "You can have data without information, but you cannot have information without data. A large amount of data generated per data, the issues in handling. How big companies manage and analyze a huge amount of data?"
```

Comment:
`A large amount of data generated per data, the issues in handling.` Is it the correct sentence @SamriddhiMishra

Reply:
@vyasriday corrected the typo. 
Thanks!